pastepwn/analyzers/emailpasswordpairanalyzer.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+from .regexanalyzer import RegexAnalyzer
+
+_EMAIL_PASSWORD_REGEX = r'[\w\.\+_-]+@[\w\._-]+\.[a-zA-Z]*\:[\w\.\+\!\$\#\^&\*\(\)\{\}\[\]\_\-\@\%\=]+$'
```

Comment:
Nice regex, but there are some characters which are often used in passwords such as `'§´/\?<>;:|",~` and the backtick character. Maybe you could add them to the regex as well? How should we deal with spaces? I would say we ignore spaces for now.

An example of a possible regex: ``\§\\\/\'\`\´\?\<\>\;\"\:\|\,\~``

Reply:
I am ignoring the spaces for now and updating the password regex

Reply:
Thank you!